prepare/metrics/normalized_sacrebleu.py

```diff
@@ -116,7 +116,7 @@
         "他方、成績評価の甘い授業が高く評価されたり、人気取りに走る教師が出たりし、成績の安売りや大学教師のレベルダウンという弊害をもたらす恐れがある、などの反省意見もある."
     ],
 ]
-task_data = len(predictions) * [{"target_language": "ja"}]
+task_data = [{"target_language": "ja"}, {"target_language": "ja"}]
```

Comment:
before the change, in eager mode, the same 'ja' served both instances, and the second instance was updated automatically once the first instance was. I first (wrongly) interpreted this as duplicated activation of the operator.